### ISSUE TITLE
Iframing, origin trials, and permissions

### ISSUE THREAD

**Issue by jfggit** [WICG/attribution-reporting-api#775]
Now that the target default permission policy for attribution reporting is `*`, I wanted to confirm the expected behavior with nesting. Specifically, iframe B inside of iframe A. After the origin trial, assuming iframe A made no changes to permission policy, we would expect the API to be available inside of iframe B with no other changes (i.e. no `allow` attribute), right?

Similarly with origin trials, assuming iframe B injects a valid token and iframe A has done nothing explicit, we expect the API to be available inside of iframe B, right? Does it make any difference if the iframes are same- or cross-origin?

**Comment by csharrison** [comment]
> Now that the target default permission policy for attribution reporting is *, I wanted to confirm the expected behavior with nesting. Specifically, iframe B inside of iframe A. After the origin trial, assuming iframe A made no changes to permission policy, we would expect the API to be available inside of iframe B with no other changes (i.e. no allow attribute), right?

Correct, assuming iframe A or the top-level document does not _explicitly_ disallow the API. By default it should be available.

> Similarly with origin trials, assuming iframe B injects a valid token and iframe A has done nothing explicit, we expect the API to be available inside of iframe B, right?

Correct.

> Does it make any difference if the iframes are same- or cross-origin?

I am actually not sure. cc @clelland to double check. I think if A and B are same origin it might be possible for A's token to be valid within B but I am not sure and it isn't clear from the documentation.
